test/node_compatibility/index.js

```diff
@@ -1,4 +1,4 @@
-const EasyPostClient = require('../..');
+const EasyPostClient = require('../../dist/easypost');
```

Comment:
Hmmm... I don't think this is the way to fix this. It's explicitly importing from the root, because thats how it will be imported in those libraries. Doing it this way would be the equivalent of
```
const EasyPostClient = require("easypost/dist/easypost");
```
that would work, but is not ideal

Reply:
Huh, when I hovered over it locally it kept telling me it was trying to import the TS file instead of the actual built lib. Is there an alternative I should look at to make it more explicit? Are you able to see the failed build errors?

Reply:
Hmmm... yea thats what I think the `.d.ts` actually does. Thats where its going to get its types from. I think the `package.json` `main` field tells it to use the index file to get the actual code. Thats how its been working for the past while.

I can look into this if you'd rather. If this version needs to go out sooner, we can just omit the "fix the build errors" part of it for now while I'm looking at it

Reply:
Ah nvm, figured it out.

Since we got rid of the mixed-name imports, it no longer needs the `.default` in the `index.js` file. Just removing that fixed it for me. 

I can push up a fix to this branch if you'd like

Reply:
Sure! Thanks for checking on this. I assume that end users won't need to change anything since they're referencing the index file ultimately, we're just changing what's referenced inside of it?

Reply:
That is correct. They are just importing the export from the index file, and don't care whats under the hood 👍

Reply:
Fantastic, mind stamping this then? 😃